src/scss/components/_menus.scss

```diff
@@ -110,7 +110,6 @@ $md-menu-cascading-list-padding: 16px;
     left: 0;
     top: 100%;
     transform-origin: 50% 0;
-    width: 100%;
```

Comment:
Maybe this should be `min-width: 100%` instead? When taking this away, do SelectField buttons still display correctly?